agent/util-scripts/unittests

```diff
@@ -255,6 +257,13 @@ declare -A options=(
     [test-31]="--prefix=foo/bar --user=ndk"
     [test-32]="--help"
     [test-34]="--help"
+    [test-35]="--group=default --start-trigger=\"START:DEFAULT\" --stop-trigger=\"STOP DEFAULT\""
+    [test-35]="--group=default --start-trigger=\"START DEFAULT\" --stop-trigger=\"STOP:DEFAULT\""
+)
+
+declare -A expected_status=(
+    [test-05]=1
+    [test-06]=1
```

Comment:
Need to add test-35 with an expected status of 1 here I believe.

Reply:
Fixed.